kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -455,6 +460,18 @@ footer {
     width: 82px;
 }
 
+/* ===================================
+    Modal Dialog Login
+=================================== */
+
+.modal-dialog {
```

Comment:
Why do we want to override all bootstrap modal dialog boxes? Is there an issue with the way that bootstrap handles the sizing?